SemanticUI/Modules/Dropdown.elm

```diff
@@ -0,0 +1,389 @@
+module SemanticUI.Modules.Dropdown exposing
+    ( Config
+    , DrawerState(..)
+    , DropdownBuilder
+    , ToggleEvent(..)
+    , dropdown
+    , init
+    , readOnly
+    , toHtml
+    , toItem
+    , toggle
+    , toggleEvent
+    )
+
+{-| Helper to create semantic-ui animated dropowns without the need for external JS.
+
+As an example a big dropdown menu using layout:
+
+    let
+        mainMenu { toDropdown, toToggle, toDrawer } =
+            toDropdown div
+                []
+                [ toToggle div [ class "text" ] [ text "File" ]
+                , toToggle i [ class "dropdown icon" ] []
```

Comment:
I'm not sure if I understand why `toToggle` is needed on child elements of toDropdown? Is it that events don't always propagate up to `toDropdown`?

Reply:
Changed this to `toToggle` on the root element. This needs some testing to confirm.

Possibly in future one could also forcibly stop on "click" event bubbling on the `drawer`, but for now I'm just doing the simplest possible thing until a reason comes up to change it.